README.md

```diff
@@ -527,6 +527,35 @@ Felizmente, há uma solução melhor para esse dilema: executar testes de contra
 
 ## <a id="sec-ui-tests"></a>UI Tests
 
+A maioria dos aplicativos tem algum tipo de interface de usuário. Normalmente, estamos falando de uma interface da Web no contexto de aplicativos da Web. Muitas vezes, as pessoas esquecem que uma API REST ou uma interface de linha de comando é tanto uma interface de usuário quanto uma interface de usuário da web sofisticada.
+
+UI tests testa se a interface do usuário do seu aplicativo funciona corretamente. A entrada do usuário deve acionar as ações corretas, os dados devem ser apresentados ao usuário, o estado da interface do usuário deve mudar conforme o esperado.
```

Comment:
Acho que seria interessante traduzir "UI tests" para "Testes de UI" ou "Testes de interface de usuário". Já "end-to-end" pode ficar sem tradução, mas a palavra "tests" faria sentido traduzir, ficando assim: testes end-to-end